workflows/config/snakemake/production_config_HomSap.yml

```diff
@@ -0,0 +1,55 @@
+"population_id": [0] # (what is this a setting for?)
+
+# snakemake settings
+"seed": 12345
+"replicates": 3
+"output_dir": "results"
+
+# species-specific settings
+"species": "HomSap"
+"demo_models": [
+    {"id":"Constant", 
+    "num_samples_per_population": [100],
+    },
+    {"id":"OutOfAfrica_3G09",
+    "num_samples_per_population": [100,100,100],
+    }
+    {"id":"OutOfAfricaArchaicAdmixture_5R19",
+    "num_samples_per_population": [100,100,100],
+    }
+]
+"genetic_map": "HapMapII_GRCh37"
+"chrm_list": "chr1, chr2, chr3, chr4, chr5, chr6, chr7, chr8, chr9, chr10, chr11, chr12, chr13, chr14, chr15, chr16, chr17, chr18, chr19, chr20, chr21"
+"dfe_list": ["none", "Gamma_H17"]
+"annotation_list": ["all_sites", "ensembl_havana_104_exons"]
+"mask_file": "workflows/masks/HapmapII_GRCh37.mask.bed"
```

Comment:
Per our conversation during coding session today, and relating to issue #74, liftover doesn't work very well for updating the old `HapmapII_GRCh37.mask.bed` file. We should instead run the python script inside the `masks` dir on the new genetic map to get the new mask_file, something like: 
`python workflows/masks/genetic_map_mask.py ~/.cache/stdpopsim/genetic_maps/HomSap/HapMapII_GRCh38 > HapmapII_GRCh38.mask.bed`

Reply:
thank you @lntran26 ! Command worked great

Reply:
I just copied what @andrewkern did 😁